pkg/v1/providers/ytt/02_addons/management-packages/addons-manager/tanzu-addons-manager_package.lib.yaml

```diff
@@ -42,10 +42,7 @@ metadata:
 spec:
   serviceAccountName: tanzu-addons-package-sa
   packageRef:
-    #! PackageName is available in TKR BOM under the particular path listed below. We do not have to validate the key existence
-    #! is because YTT will throw error if any struct is None or keys are missing, we expect to see those error and templating
-    #! will be exit immediately.
-    refName: #@ get_bom_data_for_tkr_name().addons["tanzu-addons-manager"].packageName
+    refName: addons-manager.tanzu.vmware.com
```

Comment:
i think you need to add `tkg.tanzu.vmware.com/addon-type: "packages/management-package"` annotation to all the resources in `tanzuaddonsmanagerpackageyaml` function

Reply:
annotations added